app/models/Backend.scala

```diff
@@ -214,7 +214,7 @@ class Backend @Inject() (implicit
         cursor
       )
       .map {
-        case (Seq(), _, _) => None
+        case (Seq(), _, _) => Some(KnownDrugs(0, 0, 0, 0, cursor, Seq()))
```

Comment:
@remo87 , would you mind explaining a little bit how this was having an effect on the issue?